### DISCUSSION TITLE
How do I prevent simultaneous execution of an asset?

### DISCUSSION THREAD

**Discussion by gnilrets** [dagster-io/dagster#19125] [Q&A] [answered]
I have an asset that belongs to two jobs.  The jobs can run concurrently, but the shared asset should not run concurrently with itself because it may result in data loss.  What is the best way to prevent this from happening?

**Comment by OwenKephart** [comment, marked as answer]
Hi @gnilrets -- sounds like the ability to set global asset concurrency limits is what you're looking for: https://docs.dagster.io/guides/limiting-concurrency-in-data-pipelines#limiting-opasset-concurrency-across-runs

**Reply by gnilrets** [reply]
I'm using dagster-cloud (serverless).  Do you happen to know if that uses Postgres or MySQL?

**Reply by gibsondan** [reply]
@gnilrets that feature is available in Dagster Cloud.
